configuration/pih/htmlforms/mentalHealth.xml

```diff
@@ -917,6 +917,31 @@
 
             </div>
         </div>
+
+           <div class="section-container" id="radio_button_container">
```

Comment:
this id seems inaccurate, as this isn't a radio button container, was this copied from somewhere else?

Reply:
I have created this id 'radio_button_container' because i wanted to fix the display of the field

Reply:
Ah, okay, I see why I was confused.  Element ids should always be unque, but looks like in the mental health form there are many elements with the id "radio_button_container".  Could we change this to a class, ie

<!--    My custom css -->
        .radio_button_container{
        display:flex;
        gap:50px;
        justify-content:space-between;
        }

... and then change all the id="radio_button_container" to class="radio_button container"?